### ISSUE TITLE
No public 'develop' branch

### ISSUE THREAD

**Issue by jmank88** [twitchtv/twirp#16]
[CONTRIBUTING.md](https://github.com/twitchtv/twirp/blob/master/CONTRIBUTING.md) says:
```
Twirp uses github pull requests. Fork a branch from develop, ...
```
but no develop branch exists.  Are these docs stale? Or has `develop` just not yet been pushed to the public repo?

**Comment by spenczar** [comment]
Yeah, this is just a hiccup while moving from the private repo. We should figure out the right workflow for operating in the public, I imagine it might be a little different.

**Comment by spenczar** [comment]
Fixed in b9b9e4a251050788e8cee7e12f40e1c6508f9c05.
